setup.py

```diff
@@ -25,10 +25,10 @@ def main():
         url="https://github.com/ibm/qpylib",
         packages=setuptools.find_packages(),
         install_requires=[
-            "flask>=1.1,<2",
+            "flask>=1.1",
             "requests>=2.22,<3",
             "pycryptodome>=3.9,<4",
-            "cryptography>=2.8,<4"
+            "cryptography>=3.2"
```

Comment:
The `qradar-app-base` v3 stream has cryptography `2.8`, so not sure we can bump this up:
https://ibmsecuritydocs.github.io/qradar_appfw_v2/docs/documentation/qradar_app_base_image_3.0.5_packages.html

Reply:
Good spot, thanks. I was using the v2 image's SBOM, forgot to check v3.